src/net/client/mod.rs

```diff
@@ -127,6 +141,21 @@
 //! # }
 //! ```
 
+//! # Limitations
+//!
+//! The current implementaton has the following limitations:
+//! * The [dgram] transport does not support DNS Cookies (RFC 7873,
+//!   Domain Name System (DNS) Cookies).
+//! * The [multi_stream] transport does not support timeouts or other limits on
+//!   the number of attempts to open a connection. The caller has to
+//!   implement a timeout mechanism.
+//! * The [cache] transport does not support:
+//!   * prefetching. In this context, prefetching means updating a cache entry
+//!     before it expires.
+//!   * RFC 8767 (Serving Stale Data to Improve DNS Resiliency)
```

Comment:
Are these RFC references in a particular order, and if not should they perhaps be ascending numerically?

Reply:
Also I think it would be good if RFC references are linked to the relevant IETF RFC pages.

Reply:
They are sort of sorted in order of importance. I'll add the links for the RFCs